app/experimenter/jetstream/client.py

```diff
@@ -57,12 +59,32 @@ def get_results_metrics_map(data, primary_outcome_slugs, secondary_outcome_slugs
         )
     )
 
+    if outcomes_metadata is not None:
+        BYPASS_JETSTREAM_CHECK = False
+        metrics_set_from_jetstream = set(
+            chain.from_iterable(
+                [
+                    outcomes_metadata[slug]["metrics"]
+                    + outcomes_metadata[slug]["default_metrics"]
+                    for slug in outcomes_metadata
+                ]
+            )
+        )
+    else:
+        BYPASS_JETSTREAM_CHECK = True
```

Comment:
For patterns like this another way to reduce the number of cases is treat the `else` case here as the default ie

```py
BYPASS_JETSTSREAM_CHECK = True
if outcomes_metadata is not None:
    BYPASS_JETSTREAM_CHECK = False
    ...
```

Just saves an extra clause.

Reply:
But also good idea to set this check here like this.

Oh but also, sidebar, [all caps is usually used to denote a constant](https://peps.python.org/pep-0008/#constants).